en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Click **Connector** and then select **World Bank API**.
+
+    3. In the **World Bank API Connection** window, enter `worldBankClient` as the **Connection Name** and click **Continue to Invoke API**.
+
+    4. In the **Operation** drop-down list, select **getPopulationByCountry** and enter details as follows in the other fields:
+
+        | **Field**                  | **Value**            |
+        |----------------------------|----------------------|
+        | **Country Code**           | `country`            |
+        | **Response Variable Name** | `populationByCountry`|
+
+    5. Click **Save**.
+
+    6. Now let’s extract the population value from the response, calculate the population in millions, and store it in a variable. Follow this procedure:
+
+        1. Click the last **+** icon in the low-code diagram.
+
+        2. Under **Statements**, select **Variable** and enter details as follows:
+
+            | **Field**      | **Value**                                         |
+            |----------------|---------------------------------------------------|
+            | **Type**       | `int`                                             |
+            | **Name**       | `populationMillions`                              |
+            | **Expression** | `(populationByCountry[0]?.value ?: 0) / 1000000`  |
+
+        3. Click **Save**.
+
+6. Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Under **Statements**, select **Variable** and enter details as follows:
+
+        | **Field**      | **Value**                                    |
+        |----------------|----------------------------------------------|
+        | **Type**       | `decimal`                                    |
+        | **Name**       | `totalCasesPerMillion`                       |
+        | **Expression** | `<decimal>(totalCases / populationMillions)` |
+
+    3. Click **Save**.
+
+7. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+
+    1. To build the `json` payload with data of the total cases per million in the population, add a variable.
+
+        Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
+
+         | **Field**      | **Value**                     |
+         |----------------|-------------------------------|
+         | **Type**       | `json`                        |
+         | **Name**       | `payload`                     |
+         | **Expression** | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+
+    2. Click **Save**.
+
+    3. To respond with the JSON payload, add a `Return` statement.
```

Comment:
Are these sub steps?  shall we add numbers to them? or simply add them steps in the original flow?

Reply:
The whole thing can be considered one step. We changed it this way in V1 to reduce the number of substeps